utils.py

```diff
@@ -503,6 +504,37 @@ def convert_to_str(string_to_convert):
         return string_to_convert.encode('utf-8')
     return string_to_convert
 
+def get_sorted_nodes(nodes):
```

Comment:
Backend tests should also be written for this function.

Reply:
Done!